### PR TITLE
(styles): update tailwind to v3

### PR DESCRIPTION
**Pull request by anamontiaga**
In addition to updating the latest version and plugins, the following changes have been made:

- [x] Update some specific colors (_green, yellow, purple and gray)_
- [x] Update flex-grow and flex-shrink to new naming
- [x] Remove `theme` and `variants` by default as there are no longer needed (on V3 everything is enabled by default)
- [x] Remove `purge` (is no longer needed to remove unused styles)

### CODE REVIEW

File: components/confirmation-prompt/component.tsx [hunk 1]
```diff
@@ -33,15 +33,15 @@ export const ConfirmationPrompt: FC<ConfirmationPromptProps> = ({
...
           No
         </Button>
-        <Button theme="primary" size="base" className="flex-shrink-0 sm:mr-5" onClick={onAccept}>
+        <Button theme="primary" size="base" className="shrink-0 sm:mr-5" onClick={onAccept}>
           Yes
         </Button>
 
         {icon && (
-          <Icon icon={icon} className="hidden ml-auto sm:block flex-shrink-1 flex-grow-1 w-36" />
+          <Icon icon={icon} className="flex-grow hidden ml-auto sm:block flex-shrink-1 w-36" />
```
Comment (mbarrenechea) [inline components/confirmation-prompt/component.tsx:44]:
```suggestion
          <Icon icon={icon} className="grow-1 hidden ml-auto sm:block shrink-1 w-36" />
```

**Review by mbarrenechea — CHANGES_REQUESTED** [review]
Pakages:
Don't use latest as the version to use

Colors:
Instead of changing all the classnames in the components it would be better if you add the colors inside the config

```
const colors = require('tailwindcss/colors')

module.exports = {
  theme: {
    extend: {
      colors: {
        gray: colors.slate,
        green: colors.emerald,
        yellow: colors.amber,
        purple: colors.violet,
      }
    },
  },
  // ...
}
```

**Comment by anamontiaga** [comment]
> Pakages: Don't use latest as the version to use
> 
> Colors: Instead of changing all the classnames in the components it would be better if you add the colors inside the config
> 
> ```
> const colors = require('tailwindcss/colors')
> 
> module.exports = {
>   theme: {
>     extend: {
>       colors: {
>         gray: colors.slate,
>         green: colors.emerald,
>         yellow: colors.amber,
>         purple: colors.violet,
>       }
>     },
>   },
>   // ...
> }
> ```

Why rename the colors and not adapt to the new names if we are going to use the scaffold for new projects?

**Comment by anamontiaga** [comment]
Soved in [https://github.com/Vizzuality/front-end-scaffold/pull/53](https://github.com/Vizzuality/front-end-scaffold/pull/53)

**Pull request closed without merging**